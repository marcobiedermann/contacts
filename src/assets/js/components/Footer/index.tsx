import React, { FC } from 'react';

const Footer: FC = (props) => <footer {...props} />;

export default Footer;
