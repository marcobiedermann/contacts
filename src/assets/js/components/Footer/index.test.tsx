import { render } from '@testing-library/react';
import React from 'react';
import Footer from '.';

describe('footer component', () => {
  it('renders correctly', () => {
    expect.assertions(1);

    const { asFragment } = render(<Footer />);

    expect(asFragment).toMatchSnapshot();
  });
});
