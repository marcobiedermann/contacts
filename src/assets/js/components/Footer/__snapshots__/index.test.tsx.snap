// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer component renders correctly 1`] = `[Function]`;
